### PR TITLE
New build-script for github actions.

### PR DESCRIPTION
**Pull request by eivinhb**
The old one used a ::set-env that github dropped support for (security reasons)
så we had to create a new flow. I combine the two files we now have to one
and create a better flow on the process.

(build java 1.8,11) -> find a new version -> perform release or snapshot-build

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,19 +17,50 @@ jobs:
         with:
           java-version: ${{ matrix.java }}
       - name: Build with Maven
-        run: mvn -B package --file pom.xml
+        run: mvn -B package --no-transfer-progress --file pom.xml
```
Comment (johnksv) [inline .github/workflows/build.yml:20]: I noticed we don't run tests for neither workflows. Should we?

**Review by johnksv — APPROVED** [review]
💪🏻 👏🏻  Looking good!
A minor comment.

File: .github/workflows/build.yml (hunk 1 above)
Comment (eivinhb) [reply .github/workflows/build.yml:20]: What do you mean? package include test-phase:
`Tests run: 101, Failures: 0, Errors: 0, Skipped: 0`

Comment (eivinhb) [reply .github/workflows/build.yml:20]: I could probably skip test in the release-build. But these tests are so fast, so I don't bother.

Comment (johnksv) [reply .github/workflows/build.yml:20]: Haha, ofc. Sorry about that. I blame it on not drinking any coffee yet.

**Pull request merged**